### PR TITLE
140

### PR DESCRIPTION
**Pull request by diegosanmartino**
Add mosaic tiles component

### CODE REVIEW

**Comment by seancrow** [comment]
@diegosanmartino it seems that ESLint checks have not been properly applied to the code, as expected. Please verify that and then fix the code listings accordingly.

**Comment by tdipisa** [comment]
@diegosanmartino  there is a problem with the following test case in Home-test.jsx row 36: 
'checks that homepage has been rendered'.
The Home.jsx file has been updated with a new html element having a className='header'
The test case checks the occurrences of elements with this className, so this is the reason why the test fails.

Try to run locally the following command:
npm run continuoustest

before doing a PL to see if there are errors

**Comment by diegosanmartino** [comment]
Are there others conflicts after my last commit?
commits on Oct 17, 2016 
cherry-pick:
62d576ac5e7bec247a583e59ded95be9ded9dffa
86f66c7a05c78c52cddd6aa24e065c22e1b1353a
2159cc9ae1a5975172fc078b203d40dbfa957aea

File: js/components/Mosaic.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const React = require('react');
```
Comment (diegosanmartino) [inline js/components/Mosaic.jsx]: ok

File: js/containers/Dataset.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+const {Glyphicon} = require('react-bootstrap');
+const SearchBar = require('../../MapStore2/web/client/components/mapcontrols/search/SearchBar');
+
+require('../../assets/css/home.css');
+
+const Mosaic = require('../components/Mosaic');
+
+require('../../assets/application/conoscenze_ambientali/css/skin-home.css');
+
+const Home = (props) => (
```
Comment (diegosanmartino) [inline js/containers/Dataset.jsx]: ok

File: js/containers/Dataset.jsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    </Localized>
+);
+
+module.exports = connect((state) => {
+    return {
+        error: state.loadingError || (state.locale && state.locale.localeError) || null,
+        locale: state.locale && state.locale.locale,
+        messages: state.locale && state.locale.messages || {}
+    };
+})(Home);
```
Comment (diegosanmartino) [inline js/containers/Dataset.jsx]: ok

File: js/containers/Dataset.jsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+const SearchBar = require('../../MapStore2/web/client/components/mapcontrols/search/SearchBar');
+
+require('../../assets/css/home.css');
+
+const Mosaic = require('../components/Mosaic');
+
+require('../../assets/application/conoscenze_ambientali/css/skin-home.css');
+
+const Home = (props) => (
+    <Localized messages={props.messages} locale={props.locale}>
```
Comment (diegosanmartino) [inline js/containers/Dataset.jsx]: I don't understand this comment

File: package.json [hunk 5]
```diff
@@ -141,7 +141,7 @@
   "scripts": {
     "clean": "rimraf dist",
     "compile": "npm run clean && mkdirp ./dist && webpack --config prod-webpack.config.js",
-    "start": "webpack-dev-server --progress --colors --port 8081 --hot --inline --content-base .",
+    "start": "webpack-dev-server --progress --colors --port 8082 --hot --inline --content-base .",
```
Comment (diegosanmartino) [inline package.json]: ok

File: js/components/MosaicTile.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
+const React = require('react');
```
Comment (diegosanmartino) [inline js/components/MosaicTile.jsx]: ok

File: js/containers/Dataset.jsx (hunk 4 above)
Comment (diegosanmartino) [inline js/containers/Dataset.jsx]: I removed Localized

File: js/components/Mosaic.jsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    getDefaultProps() {
+        return {
+            tiles: []
+        };
+    },
+
+    renderTiles() {
+        return this.props.tiles.map(function(tile) {
+            return (<Tile
+                        setData {...tile}
```
Comment (diegosanmartino) [inline js/components/Mosaic.jsx]: removed

File: js/containers/App.jsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
+/**
```
Comment (diegosanmartino) [inline js/containers/App.jsx]: mmm no? I removed App.jsx

**Pull request merged**